packages/background/src/keyring/keyring.ts

```diff
@@ -200,10 +207,6 @@ export class KeyRing {
       : defaultCoinType;
   }
 
-  public getKeyFromCoinType(coinType: number): Key {
```

Comment:
Is there a reason you deleted this method? `getKeyFromCoinType` method has different roles with `getKey` . For networks like Secret, they have an alternative bip44 path and if a user has multiple accounts that accounts for the bip44 path, we allow users to select their account to use. `getKey` allows prioritizing and selecting coin type based on the bip44 path the user selected while `getKeyFromCoinType` returns the requested coin type account regardless of that. `getKeyFromCoinType` is needed to allow users to select bip44 path.

Reply:
That was my mistake, I did not catch the difference. Thanks for pointing that out.

I've since updated it to include the Ethereum address property for the chain; while it will not allow one to differentiate between Ethereum addresses for each path, it will allow one apply Ethereum address generation to all paths, which seems to be consistent with this PR. Let me know if I'm missing anything.